.claude/metadata-updater.sh

```
@@ -0,0 +1,136 @@
+fi
+
+# Update a single key in metadata
+update_metadata_key() {
+  local key="$1"
+  local value="$2"
+
+  # Create temp file
+  local temp_file="${metadata_file}.tmp"
+
+  # Escape special sed characters in value (& | / \)
+  local escaped_value=$(echo "$value" | sed 's/[&|\/]/\\&/g')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)